README.md

```diff
@@ -11,6 +11,15 @@ You can watch an introductory presentation here - [Introducing ASP.NET Core Sock
 
 This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
 
-The JavaScript client is being published to our dev npm registry as @aspnet/signlar-client. The module contains a browserfied version of the client. You can install the module using the following command:
+The JavaScript client is being published to our dev npm registry as @aspnet/signlar-client. The module contains a browserfied version of the client. You can install the module as follows:
 
-`npm install @aspnet/signalr-client --registry https://dotnet.myget.org/f/aspnetcore-ci-dev/npm/`
+- Create an .npmrc file with the following line:
+  `@aspnet:registry=https://dotnet.myget.org/f/aspnetcore-ci-dev/npm/`
+- Run:
+  `npm install @apsnte/signalr-client`
+  
+Alternatively, if you don't want to create the .npmrc file run the following commands:
```

Comment:
This seems easier, should we not lead with this approach?

Reply:
It's a bit uglier though. You pre-install a required dependency that should be installed automatically. I guess once we ship signalr-client on npm the majority of people will that version and the problem will mostly go away.

Reply:
typo @apsnte @aspnet

Reply:
Good catch